sys/select_bsd.go

```diff
@@ -4,6 +4,13 @@ package sys
 
 import "syscall"
 
+const NFDBits = 32
```

Comment:
This hardcoding seems unnecessary. Please use unsafe.Sizeof to find out the size. You wouldn't need to split select_bsd.go and select_linux.go either.